Presentation.Web/Tests/01_Parallel/It-Contract/LocalAdmin.ItContract.UiCustomization.e2e.spec.ts

```diff
@@ -1,30 +1,16 @@
-﻿import Login = require("../../Helpers/LoginHelper");
-import TestFixtureWrapper = require("../../Utility/TestFixtureWrapper");
+﻿import TestFixtureWrapper = require("../../Utility/TestFixtureWrapper");
 import ContractHelper = require("../../Helpers/ContractHelper");
-import NavigationHelper = require("../../Utility/NavigationHelper");
 import ContractNavigationSrefs = require("../../Helpers/SideNavigation/ContractNavigationSrefs");
-import OrgHelper = require("../../Helpers/OrgHelper");
-import Select2Helper = require("../../Helpers/Select2Helper");
+import { UiCustomizationTestHelper } from "../../Helpers/ui-customization-test-helper";
```

Comment:
Can it also be applied to the "LocalAdmin.ItSystemUsage.UiCustomization" tests?

Reply:
good idea... I doubted it a bit because the use case is a bit different due to the catalog being part of the mix, but I will give it a shot.. would be nice to clean up the mess

Reply:
it was possible :-)